source/stylesheets/library/_variables.scss

```diff
@@ -0,0 +1,43 @@
+// Typography
+$base-font-family: "freight-text-pro", $font-stack-georgia;
```

Comment:
Since we're not using Freight in our brand anymore and this is departed from thoughtbot branding anyway, thoughts on just leaving these to both the system stack?